spigot-tests/index.js

```diff
@@ -1,20 +1,44 @@
 const { Spigot } = require('genesyscloud-spigot/dist/src/index');
 
+let envConfig = {};
+
+const defaultDevConfig = {
+  OAUTH_CLIENT_ID: 'ff22e32c-2948-4ff4-8f2c-1c379d28e84d',
+  ORG: 'TEST-valve-1ym37mj1kao',
+  USERNAME: 'agent-7-1ym37mj1kao@example.com',
+  PASSWORD: 'fV-qIe4HZtGM1yLr}^',
+  ENV_HOST: 'inindca.com'
+};
```

Comment:
Is it ok to have these in Github? I know it is just dca, but it still feels wrong.